example/prometheus-operator-crd-full/monitoring.coreos.com_prometheusagents.yaml

```diff
@@ -4016,14 +4016,15 @@ spec:
                 x-kubernetes-map-type: atomic
               podMonitorSelector:
                 description: "*Experimental* PodMonitors to be selected for target
-                  discovery. \n If `spec.serviceMonitorSelector`, `spec.podMonitorSelector`
-                  and `spec.probeSelector` are null, the Prometheus configuration
-                  is unmanaged. The Prometheus operator will ensure that the Prometheus
-                  configuration's Secret exists, but it is the responsibility of the
-                  user to provide the raw gzipped Prometheus configuration under the
-                  `prometheus.yaml.gz` key. This behavior is deprecated and will be
-                  removed in the next major version of the custom resource definition.
-                  It is recommended to use `spec.additionalScrapeConfigs` instead."
+                  discovery. \n If `spec.serviceMonitorSelector`, `spec.podMonitorSelector`,
+                  `spec.ProbeSelector` and `spec.scrapeConfigSelector` are null, the
```

Comment:
Should this be ```suggestion
                  `spec.probeSelector` and `spec.scrapeConfigSelector` are null, the
```? (i think there are multiple occurrences)

Reply:
Good catch! I just rebased against main and pulled PrometheusAgent in, fixing those :+1:

Reply:
And actually it was in CommonPrometheusFields, fixed in c577070.